### PR TITLE
Style/gdh 929 card animation

### PR DESCRIPTION
**Pull request by robbieatacato**
Purpose:

- animate news card: bem, css, tokens
- animate dynamic content card: bem, css, tokens

Figma:
- https://www.figma.com/proto/qr18lJ6PZP8HMAN77HBxCp/Schetsboek---Transitie-Online?page-id=686%3A234006&node-id=1461%3A305807&viewport=1195%2C1143%2C0.12&scaling=min-zoom&starting-point-node-id=1385%3A301248&show-proto-sidebar=1&hide-ui=1

Screen capture:
https://user-images.githubusercontent.com/95216123/204296089-f56f9772-c650-4bc8-bd37-6bc47c1f4fc7.mov

#closes #1151

### CODE REVIEW

File: components/CardNews/src/index.scss [hunk 1]
```diff
@@ -68,9 +105,32 @@
...
+}
+
+.denhaag-card-news--hover .denhaag-card-news__icon,
+.denhaag-card-news:hover .denhaag-card-news__icon {
+  opacity: var(--denhaag-card-news-icon-opacity);
+}
+
+.denhaag-card-news__icon::before {
+  content: "";
+  background-color: var(--denhaag-color-blue-3);
```
Comment (paulAcato) [inline components/CardNews/src/index.scss:126]: Sort alphabetical please (our code-standards).

File: components/CardNews/src/index.scss [hunk 2]
```diff
@@ -18,39 +18,76 @@
...
 }
 
 .denhaag-card-news__image {
   aspect-ratio: var(--denhaag-card-news-image-aspect-ratio);
-  height: var(--denhaag-card-news-image-height);
+  height: inherit;
   object-fit: var(--denhaag-card-news-image-object-fit);
-  width: var(--denhaag-card-news-image-width);
+  width: inherit;
+  transition: all var(--denhaag-card-news-transition-timing-function, ease-in-out)
```
Comment (paulAcato) [inline components/CardNews/src/index.scss:63]: Target specific changes. Now you make the transition heavier for no reason.

File: components/CardNews/src/index.scss [hunk 3]
```diff
@@ -18,39 +18,76 @@
...
+.denhaag-card-news--hover .denhaag-card-news__image,
+.denhaag-card-news:hover .denhaag-card-news__image {
+  transform: var(--denhaag-card-news-translate) scale(1.02);
+}
+
+.denhaag-card-news .denhaag-icon {
+  --denhaag-card-news-icon-translate: translateX(-0.1875rem);
+
+  transition: all var(--denhaag-card-news-transition-timing-function, ease-in-out)
+    var(--denhaag-card-news-transition-duration, 200ms);
```
Comment (paulAcato) [inline components/CardNews/src/index.scss:76]: Target specific changes. Now you make the transition heavier for no reason.

File: components/DynamicContent/src/index.scss [hunk 4]
```diff
@@ -28,15 +28,28 @@
 }
 
 .denhaag-dynamic-content__card {
+  --denhaag-dynamic-content-card-icon-opacity: 0;
+
   display: flex;
   flex-direction: column;
   gap: var(--denhaag-dynamic-content-card-gap);
   position: relative;
+  transition: all var(--denhaag-dynamic-content-card-transition-timing-function, ease-in-out)
```
Comment (paulAcato) [inline components/DynamicContent/src/index.scss:37]: Target specific changes. Now you make the transition heavier for no reason.

File: components/DynamicContent/src/index.scss [hunk 5]
```diff
@@ -52,10 +65,27 @@
...
+  height: var(--denhaag-dynamic-content-image-container-height);
+  width: var(--denhaag-dynamic-content-image-container-width);
+  overflow: hidden;
+}
+
 .denhaag-dynamic-content__card-image {
   aspect-ratio: 16 / 9;
   background: var(--denhaag-dynamic-content-image-background-color);
   object-fit: cover;
+  transition: all var(--denhaag-dynamic-content-card-transition-timing-function, ease-in-out)
```
Comment (paulAcato) [inline components/DynamicContent/src/index.scss:78]: Target specific changes. Now you make the transition heavier for no reason.

File: components/DynamicContent/src/index.scss [hunk 6]
```diff
@@ -66,8 +96,18 @@
 }
 
 .denhaag-dynamic-content__card .denhaag-icon {
+  --denhaag-dynamic-content-card-icon-translate: translateX(-0.1875rem);
+
   align-self: end;
   color: var(--denhaag-dynamic-content-icon-color);
+  transition: all var(--denhaag-dynamic-content-card-transition-timing-function, ease-in-out)
```
Comment (paulAcato) [inline components/DynamicContent/src/index.scss:103]: Target specific changes. Now you make the transition heavier for no reason.

File: components/DynamicContent/src/index.scss [hunk 7]
```diff
@@ -52,10 +65,27 @@
...
 .denhaag-dynamic-content__card-image {
   aspect-ratio: 16 / 9;
   background: var(--denhaag-dynamic-content-image-background-color);
   object-fit: cover;
+  transition: all var(--denhaag-dynamic-content-card-transition-timing-function, ease-in-out)
+    var(--denhaag-dynamic-content-transition-card-duration, 200ms);
+}
+
+.denhaag-dynamic-content__card--reduced-motion .denhaag-dynamic-content__card-image {
+  transition: none;
```
Comment (paulAcato) [inline components/DynamicContent/src/index.scss:83]: keep using CSS variables.

File: components/DynamicContent/src/index.scss [hunk 8]
```diff
@@ -117,6 +158,34 @@
   word-break: break-word; // Fixes hypens issue for Chrome & safari on windows device.
 }
 
+.denhaag-dynamic-content__card-icon {
+  opacity: var(--denhaag-dynamic-content-icon-opacity);
+  height: var(--denhaag-dynamic-content-icon-size, var(--denhaag-space-block-xl));
+  width: var(--denhaag-dynamic-content-icon-size, var(--denhaag-space-block-xl));
+  color: var(--denhaag-dynamic-content-icon-color);
+  position: relative;
+  transition: all var(--denhaag-dynamic-content-card-transition-timing-function, ease-in-out)
```
Comment (paulAcato) [inline components/DynamicContent/src/index.scss:167]: Target specific changes. Now you make the transition heavier for no reason. And sort alphabetically

File: components/DynamicContent/src/index.scss [hunk 9]
```diff
@@ -117,6 +158,34 @@
...
+    var(--denhaag-dynamic-content-card-transition-duration, 200ms);
+  align-self: end;
+}
+
+.denhaag-dynamic-content__card-icon::before {
+  content: "";
+  background-color: var(--denhaag-color-blue-3);
```
Comment (paulAcato) [inline components/DynamicContent/src/index.scss:174]: Sort alphabetically conform our frontend code-standards

File: components/CardNews/src/index.scss [hunk 10]
```diff
@@ -18,39 +18,76 @@
   width: var(--denhaag-card-news-width);
   height: var(--denhaag-card-news-height);
   position: relative;
+  transition: all var(--denhaag-card-news-transition-timing-function, ease-in-out)
```
[the remark this answers is not in the document]
Comment (robbieatacato) [reply components/CardNews/src/index.scss:20]: This is referencig two transitions:
Screenshot 2022-11-29 at 16 40 17

I cannot put these two in the shorthand 'transition:' also its not possible to do twice this transition in the selector.

File: components/CardNews/src/index.scss [hunk 11]
```diff
@@ -68,9 +105,32 @@
 }
 
 .denhaag-card-news__icon {
-  height: var(--denhaag-card-news-icon-height);
-  text-align: var(--denhaag-card-news-icon-text-align);
+  opacity: var(--denhaag-card-news-icon-opacity);
```
[the remark this answers is not in the document]
Comment (robbieatacato) [reply components/CardNews/src/index.scss]: done

File: components/DynamicContent/src/index.scss (hunk 6 above)
Comment (robbieatacato) [reply components/DynamicContent/src/index.scss:103]: This is referencing more than one transitions:
Screenshot 2022-11-29 at 17 14 45
Screenshot 2022-11-29 at 17 15 04

I cannot put these in the shorthand 'transition:' also its not possible to do twice this transition in the selector.

File: components/CardNews/src/index.scss (hunk 1 above)
Comment (robbieatacato) [reply components/CardNews/src/index.scss:125]: done

File: components/DynamicContent/src/index.scss (hunk 9 above)
Comment (robbieatacato) [reply components/DynamicContent/src/index.scss:173]: done

File: components/CardNews/src/index.scss (hunk 10 above)
[the remark this answers is not in the document]
Comment (robbieatacato) [reply components/CardNews/src/index.scss]: done

File: components/CardNews/src/index.scss (hunk 2 above)
Comment (robbieatacato) [reply components/CardNews/src/index.scss]: done

File: components/CardNews/src/index.scss (hunk 3 above)
Comment (robbieatacato) [reply components/CardNews/src/index.scss]: done

File: components/DynamicContent/src/index.scss (hunk 5 above)
Comment (robbieatacato) [reply components/DynamicContent/src/index.scss]: done

File: components/DynamicContent/src/index.scss (hunk 6 above)
Comment (robbieatacato) [reply components/DynamicContent/src/index.scss]: done

**Review by paulAcato — APPROVED** [review]

**Pull request merged**